File: BankApplicationAPI/BankApplicationAPI/Models/LoanPaymentSchedule.cs
namespace BankApplicationAPI.Models;

public partial class LoanPaymentSchedule
{
    public int PaymentId { get; set; }

    public int? LoanId { get; set; }

    public DateTime? PaymentDate { get; set; }

    public decimal? PaymentAmount { get; set; }

    public decimal? BalanceAfterPayment { get; set; }

    public string? PaymentStatus { get; set; }

    public virtual LoanApplication? Loan { get; set; }
}

File: BankApp/src/app/Modules/LoanPaymentSchedule.ts
import { LoanApplication } from "./LoanApplication";

export interface LoanPaymentSchedule {
    paymentId?: number | null;
    loanId?: number | null;
    paymentDate?: Date | null;
    paymentAmount?: number | null;
    balanceAfterPayment?: number | null;
    paymentStatus?: string | null;
    loan?: LoanApplication | null;
}